user_guide/intro_users_and_benefits.rst

```diff
@@ -0,0 +1,163 @@
+.. _oro-benefits:
+
+OroCRM Users and Their Benefits
+===============================
+
+This chapter of the guide explains the most common OroCRM user roles in order to help you understand which sections of 
+this user guide best match your work duties.
+
+Something for Everyone
+----------------------
+
+Collect and Process information easily
```

Comment:
That really depends on Oro's preference for this guide. There is no standard for title (Most of the Words Are Capitalized) vs. sentence capitalization (Only the first word is capitalized) on headers/sub-headers. However, they should be consistent throughout the article.

Also important to note: AP style capitalizes all words with four or more letters, including "with." If you're unsure, I often use http://titlecapitalization.com/ to double check.